rollup/internal/orm/batch.go

```diff
@@ -226,7 +222,7 @@ func (o *Batch) GetBatchByIndex(ctx context.Context, index uint64) (*Batch, erro
 }
 
 // InsertBatch inserts a new batch into the database.
-func (o *Batch) InsertBatch(ctx context.Context, batch *encoding.Batch, dbTX ...*gorm.DB) (*Batch, error) {
+func (o *Batch) InsertBatch(ctx context.Context, batch *encoding.Batch, useCodecv0 bool, dbTX ...*gorm.DB) (*Batch, error) {
```

Comment:
I think it would a bit more elegant and readable to pass an enum instead of passing a bool.

Reply:
changed to enum in [eb74b18](https://github.com/scroll-tech/scroll/pull/1201/commits/eb74b183541b02ba07772032ec07abda04281aef)